### PR TITLE
Kepler fix to development

### PR DESCRIPTION
**Pull request by olaszsoma**
Kepler crash fix should be merged into development.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,8 +18,8 @@ if($ENV{ITM_ENVIRONMENT_LOADED})
 	message("###Compiler set to ITM###")
 	
 	#gcov flags
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mno-avx  -fprofile-arcs -ftest-coverage")
-	message("-- Coverage flags set")
+	#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mno-avx  -fprofile-arcs -ftest-coverage")
```
Comment (gergopokol) [inline CMakeLists.txt]: To be checked

File: CMakeLists.txt [hunk 2]
```diff
@@ -18,8 +18,8 @@ if($ENV{ITM_ENVIRONMENT_LOADED})
...
+	#message("-- Coverage flags set")
```
Comment (gergopokol) [inline CMakeLists.txt]: To be checked

**Review by gergopokol — CHANGES_REQUESTED** [review]
CMakeLists to be cheched

**Comment by pordaniel01** [comment]
Removed commenting that removed gcov analsis.

File: CMakeLists.txt (hunk 2 above)
Comment (pordaniel01) [reply CMakeLists.txt]: Uncommented

File: CMakeLists.txt (hunk 1 above)
Comment (pordaniel01) [reply CMakeLists.txt]: Uncommented

**Review by gergopokol — APPROVED** [review]
Tested by Dani.

**Pull request merged**